feature-libs/asm/components/customer-selection/customer-selection.component.ts

```diff
@@ -42,6 +44,9 @@ export class CustomerSelectionComponent implements OnInit, OnDestroy {
 
   @ViewChild('resultList') resultList: ElementRef;
   @ViewChild('searchTerm') searchTerm: ElementRef;
+  @ViewChildren('resultItem') resultItems: QueryList<ElementRef<HTMLElement>>;
```

Comment:
The name for this variable can be more descriptive i.e. `searchResultItems`.

Reply:
ok will update